app/views/onboarding/unauthorized.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= c 'onboarding_response',
+  style: :error,
+  heading: I18n.t('components.onboarding_unauthorized.heading'),
+  text: I18n.t('components.onboarding_unauthorized.text').html_safe
+%>
```

Comment:
@mattwr18: We would need to make this configurable using the settings UI as well. If you want to, I can add the necessary fields.

Reply:
done